package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "traxercise",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview --port 4173",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
+  },
+  "dependencies": {
```

Comment:
please remove the unnecessary dependencies and devDependencies. There are three kind of datepicker libraries. Please review all of the libraries you need